riff-raff/conf/routes

```diff
@@ -119,6 +119,7 @@ GET         /testing/view/:uuid                             controllers.Testing.
 GET         /testing/addStringUUID                          controllers.Testing.transferAllUUIDs
 GET         /testing/testcharset                            controllers.Testing.testcharset
 GET         /testing/deployinfo                             controllers.Testing.hosts
+GET         /testing/doHousekeeping                         controllers.Testing.doHousekeeping
```

Comment:
This probably wants to be removed before release 😄

Reply:
Yes. Good point. Will remove it once I've finished testing.